FabNESO/tasks.py

```diff
@@ -252,3 +265,309 @@ def neso_ensemble(
         )
         fab.with_config(config)
         fab.run_ensemble(config, sweep_dir)
+
+
+def _parse_vbmc_bounds_string(
+    vbmc_bounds_string: str,
+    delimiter: str,
+) -> tuple[float, float, float, float]:
+    (
+        lower_bound,
+        upper_bound,
+        plausible_lower_bound,
+        plausible_upper_bound,
+    ) = vbmc_bounds_string.split(delimiter)
+    return (
+        float(lower_bound),
+        float(upper_bound),
+        float(plausible_lower_bound),
+        float(plausible_upper_bound),
+    )
+
+
+@fab.task
+@fab.load_plugin_env_vars("FabNESO")
+def neso_vbmc(
+    config: str,
+    solver: str = "Electrostatic2D3V",
+    conditions_file_name: str = "conditions.xml",
+    mesh_file_name: str = "mesh.xml",
+    processes: int = 4,
+    nodes: int = 1,
+    cpus_per_process: int = 1,
+    wall_time: str = "00:15:00",
+    output_directory_name: str = "",
+    reference_field_file: str = "",
```

Comment:
Rather than making this default to an empty string, I'd say probably better to not give a default here and have this as a required argument as that might suggest it doesn't need to be set.